_assets/scss/elements/_typography.scss

```diff
@@ -78,6 +78,17 @@
   @include breakpoint($large) {}
 }
 
+@mixin paragraph {
```

Comment:
I don't think we need a paragrah mixin because it is essentially an H4 Headline.  I want to follow the design requirements of this https://www.dropbox.com/sh/1e87f8cogkrxfom/AADQ0bf41pK6KpevnjqlpsgEa/_Manual/Type%20ramp?dl=0&preview=R+web+type+ramp+v02.jpg as closely as possible using them same naming so that we can speak the same language with Sean and Andre.